src/lib/events.js

```diff
@@ -44,7 +44,7 @@ export default () => {
     }
 
     if (_shared.state === 'refreshing') {
-      if (_el.shouldPullToRefresh() && _shared.pullStartY < _shared.pullMoveY) {
+      if (_el.shouldPullToRefresh() && _shared.pullStartY < _shared.pullMoveY && e.cancelable) {
```

Comment:
I would perform the check for `e.cancelable` first, that way we don't make unnecessary work before more checks.